.github/workflows/discussion-triage.yml

```diff
@@ -0,0 +1,28 @@
+name: Discussion Triage
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'discuss'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create a discussion in github/cli
+        run: |
+            export TITLE="Triage: $(gh issue view ${ISSUE_NUMBER} --json title --template {{.title}}) (#${ISSUE_NUMBER})"
+            gh api graphql -f query='
+              mutation($title: String!, $body: String!) {
+                createDiscussion(
+                    input: {repositoryId: "R_kgDODAqwpw", categoryId: "DIC_kwDODAqwp84CajJa", title: $title, body: $body}
+                ) {
+                    discussion {
+                      id
+                    }
+                  }
+              }' -f title="${TITLE}" -f body="@${LABELLER} added the discuss label to #${ISSUE_NUMBER}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_CLI_DISCUSSION_TRIAGE_TOKEN }}
```

Comment:
This hasn't been created yet, I hope we could do a finely grained access token but I don't know for sure.

Reply:
**Should you event need a PAT or a GitHub App to create a discussion within the same repository?**

According to https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs, I'd assume you just need to give the automatic token the `discussions` scope and voila.

Reply:
See https://github.com/cli/cli/pull/8264#discussion_r1376282236

Reply:
Gotcha, thanks.